Makefile

```diff
@@ -563,6 +573,13 @@ failure_tests:=$(wildcard tests/failing/*.json)
 
 test-failure: $(failure_tests:=.run-expected)
 
+# kevm_pyk Tests
+
+kevm_pyk_tests := tests/specs/examples/erc20-bin-runtime.k \
+                  tests/specs/examples/vat-bin-runtime.k
+
+test-kevm-pyk: $(kevm_pyk_tests)
```

Comment:
I guess we should test more than the fact that we're generating them successfully, rather that:

- We generate them, and they are textually identical to `*.expected` files we see.
- They are `kompile`-able.

Reply:
So you can modify it (to check that they are identical to expected files):

```
test-kevm-pyk: $(kevm_pyk_tests:=.gen)

tests/specs/examples/%.k.gen: tests/specs/examples/%.k
    $(CHECK) $< $<.expected
```

It may be tougher to check that they are kompilable, but we already are testing it at least for the ERC20 example because we're running some proofs with the generated file. So that may be enough for now.